koswat/cost_report/profile/quantity_cost_parameters.py

```diff
@@ -111,23 +111,22 @@ def get_material_total_quantity_parameters(
                 self.new_core_volume.total_cost,
                 self.new_core_volume.total_cost_with_surtax,
             )
-        elif material_type == KoswatMaterialType.CLAY:
+        if material_type == KoswatMaterialType.CLAY:
             if not self.new_clay_volume:
                 return math.nan, math.nan
             return (
                 self.new_clay_volume.total_cost,
                 self.new_clay_volume.total_cost_with_surtax,
             )
-        elif material_type == KoswatMaterialType.GRASS:
+        if material_type == KoswatMaterialType.GRASS:
             if not self.new_grass_volume:
                 return math.nan, math.nan
             return (
                 self.new_grass_volume.total_cost,
                 self.new_grass_volume.total_cost_with_surtax,
             )
-        else:
-            raise ValueError(
-                "Material {} currently not supported.".format(
-                    material_type.name.capitalize()
-                )
+        raise ValueError(
+            "Material {} currently not supported.".format(
+                material_type.name.capitalize()
             )
+        )
```

Comment:
Not sure why are you determined to change the `elif` statements to `if'` without really being needed.

Reply:
See other comment